bitnami/dataplatform-bp2/Chart.yaml

```diff
@@ -0,0 +1,59 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.0.0
+dependencies:
+  - condition: kafka.enabled
+    name: kafka
+    repository: https://charts.bitnami.com/bitnami
+    version: 12.x.x
+  - condition: spark.enabled
+    name: spark
+    repository: https://charts.bitnami.com/bitnami
+    version: 5.x.x
+  - condition: elasticsearch.enabled
+    name: elasticsearch
+    repository: https://charts.bitnami.com/bitnami
+    version: 14.x.x
+  - condition: logstash.enabled
+    name: logstash
+    repository: https://charts.bitnami.com/bitnami
+    version: 3.x.x    
+  - condition: zookeeper.enabled
```

Comment:
ZooKeeper is only used in Kafka, right? Would it make more sense to use the ZooKeeper as subchart of Kafka by setting `kafka.zookeeper.enabled` to `true` (the default value)?

Reply:
Yes, I will make this change.